### ISSUE TITLE
How do I get the EPUB classic design back?

### ISSUE THREAD

**Issue by wolfinston85** [sumatrapdfreader/sumatrapdf#3561]
I updated my Sumatra recently after having to reformat my pc and noticed my EPUB files look like this:

![new version](https://github.com/sumatrapdfreader/sumatrapdf/assets/62920423/a9eb63e9-45f9-46e4-b253-bb6c8a1172e4)

I tried to use a previous version (3.3.3) based on this post I found [https://github.com/sumatrapdfreader/sumatrapdf/discussions/2712#:~:text=the%20problem%27s%20fixed-,Edit%3A%20Fixed.,-SumatraPDF%2D3.3.3%2D64](url)

By comparison that version looks like this: 

![classic version](https://github.com/sumatrapdfreader/sumatrapdf/assets/62920423/634c1959-33a0-45dc-b957-6956c1037372)

So is there any way to return to the old version without having to downgrade?

Thanks in advance.

**Comment by GitHubRulesOK** [comment]
no

**Comment by wolfinston85** [comment]
Aw, that's too bad, but as long as I can keep the old version I guess I'm happy; also, I'm sorry but did you mean "**we'll** keep an eye on that open issue"? Or were you talking about the link I posted not working?

I checked the link, not sure why it didn't work the way I formatted it. Either way, here's the discussion I was talking about:

https://github.com/sumatrapdfreader/sumatrapdf/discussions/2712
